### PR TITLE
Remove cipher suite not supported by Java

### PR DESCRIPTION
**Pull request by hmusum**
TLS_CHACHA20_POLY1305_SHA256 is not supported by Java, see https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/master/src/java.base/share/classes/sun/security/ssl/CipherSuite.java#L483

@bjorncs FYI

### CODE REVIEW

File: security-utils/src/main/java/com/yahoo/security/tls/TlsContext.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
  */
 public interface TlsContext extends AutoCloseable {
 
+    // TODO: Where does this set come from?
```
Comment (vekterli) [inline security-utils/src/main/java/com/yahoo/security/tls/TlsContext.java]: See https://github.com/vespa-engine/vespa/blob/master/vespalib/src/vespa/vespalib/net/tls/impl/iana_cipher_map.cpp#L16-L20

**Review by freva — APPROVED** [review]
So do need to update `iana_cipher_map.cpp` too then?

**Comment by vekterli** [comment]
I don't think thats necessary since ciphers will be negotiated for each connection between any C++ and Java components

File: security-utils/src/main/java/com/yahoo/security/tls/TlsContext.java (hunk 1 above)
Comment (hmusum) [reply security-utils/src/main/java/com/yahoo/security/tls/TlsContext.java]: Thanks, will resolve TODO in  another PR

**Pull request merged**